### DISCUSSION TITLE
What is the differences between Map table and distance table?

### DISCUSSION THREAD

**Discussion by Syab05** [acorg/Racmacs#161] [Q&A] [answered]
HI! I want to know what is the differences between map table and distance table. Further, the stress table.
What is actually meant by stress table? is it only for antigen-serum pairs distance?

Can we have stres table from Ag-Ag distance?

**Comment by drserajames** [comment]
The stress table computes the contribution to the map stress from each antigen-serum pair. The stress depends on the distance (calculated from titres) from the table [table distance] and optimised distance from the map [map distance]. As there is no titre measurements between antigens, there can be no stress calculation for antigen-antigen pairs.

Hope that helps!

**Reply by Syab05** [reply]
Thanks!

**Comment by shwilks** [comment, marked as answer]
You may find it helpful to read [An introduction to antigenic cartography](https://acorg.github.io/Racmacs/articles/intro-to-antigenic-cartography.html)

I have also opened discussions for Racmacs which is a better place for these types of questions

**Reply by Syab05** [reply]
Thanks!
